### ISSUE TITLE
avis home page ! Selection of date in begin date picker is not announced.

### ISSUE THREAD

**Issue by Kishan1601** [Kishan1601/Test-1#1]
Environment
UAT 2022-03
Windows 10
Chrome Version 98.0.4758.102 (Official Build) (64-bit)
NVDA 2021.3.5

Repro Steps
1. Open the URL https://www.avis.com/en/home
2. Tab to the begin date picker option
3. Press Up/Down arrow keys to navigate in calendar

Expected Result 
Change of date in date picker using arrow keys should be announce by the screen reader

Actual Result
In begin date picker user is unable to navigate and select a date in calendar using arow keys as the screen reader does not announce the change of date using arrow keys

WCAG Guideline
https://www.w3.org/TR/WCAG21/#keyboard
